_includes/layout/base/pagetop-donation.html

```diff
@@ -0,0 +1,71 @@
+{% comment %}
+This file is licensed under the MIT License (MIT) available on
+http://opensource.org/licenses/MIT.
+{% endcomment %}
+
+<div class="donation-container">
+  <div class="donation-text">
+    <p>
+      {% translate donation-banner-text layout %}
+    </p>
+    <button class="donation-btn" onclick="openDonationModal()">
```

Comment:
`type="button"` is needed in the forms, otherwise it will default to a submit type and submit a form. In the modal `<form>` is not needed, thus not used, therefore `type="button"` not needed either.